index.ts

```diff
@@ -1,3 +1,7 @@
-import { buildCdnFromAppConfig } from "./src";
-  
-fly.http.respondWith(buildCdnFromAppConfig());
```

Comment:
This change needs to be reverted, we actually need the `index.ts` to stay as is for now.